src/app.js

```diff
@@ -1 +1,39 @@
+/* eslint-disable no-console */
 'use strict';
+
+const rl = require('readline');
+const { generateNumber } = require('./generateNumber');
+const { calculating } = require('./calculating');
+
+const number = generateNumber();
+
+const terminal = rl.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+function playGame(startMessage) {
+  terminal.question(startMessage, (answer) => {
+    const { bulls, cows } = calculating(number, answer);
+
+    if (answer.length !== 4) {
+      console.log(`Should be a 4-digit number, you entered ${answer.length}\n`);
+      playGame('Try again: \n');
+
+      return;
+    }
+
+    console.log(`Bulls: ${bulls}, Cows: ${cows} \n`);
+
+    if (bulls.length !== number.length) {
+      playGame(`Guess the ${cows.length} digit numbers? \n`);
```

Comment:
The message string for starting the game can be placed in a separate variable or function (if you want to dynamically show how many numbers are left to guess) and reuse it.